images/Images/ImageConcat.tcc

```diff
@@ -487,7 +488,34 @@ Bool ImageConcat<T>::doGetMaskSlice (Array<Bool>& buffer,
 template <class T>
 IPosition ImageConcat<T>::doNiceCursorShape (uInt maxPixels) const
 {  
-   return latticeConcat_p.niceCursorShape(maxPixels);
+  
+  //Return the smallest cursor shape along the non X-Y direction from the constituent images
+  if(nimages() > 0){
+    ///if image has 1 or  2 axes return cursor shape of first image
+    if(shape().nelements() <= 2)
```

Comment:
A multi-line if or for without braces is considered bad practice.
A single line if/for is also bad, since it makes debugging and code coverage tests harder.